servicetalk-transport-netty-internal/src/testFixtures/java/io/servicetalk/transport/netty/internal/ExecutionContextExtension.java

```diff
@@ -116,6 +116,13 @@ public static ExecutionContextExtension single(NettyIoThreadFactory nettyIoThrea
         return fixed(1, nettyIoThreadFactory);
     }
 
+    /**
+     * Set to true if the extension is being shared among all tests in a class.
+     *
+     * @param classLevel true if extension is shared between tests within test class otherwise false to create a new
```

Comment:
consider rephrasing the last part of the sentence. as it is, it reads as if it creates instances itself. 
e.g. `true if extension is shared between tests within test class otherwise false to indicate that the extension's lifecycle should be follow per-test behavior`